bundle/Resources/public/js/views/plugins/cof-selectcontenttypeplugin.js

```diff
@@ -16,6 +16,70 @@ YUI.add('cof-selectcontenttypeplugin', function (Y) {
     Y.cof.Plugin.SelectContentType = Y.Base.create('selectContentType', Y.eZ.Plugin.ContentCreate, [], {
         initializer: function () {
             this.onHostEvent('*:fetchContentTypes', this._getContentTypes, this);
+            this.onHostEvent('*:fetchSuggestedLocations', this._getSuggestedLocations, this);
+        },
+
+        /**
+         * Get the suggested locations
+         *
+         * @protected
+         * @method _getSuggestedLocations
+         * @param event {Object} event facade
+         */
+        _getSuggestedLocations: function (event) {
+            var contentTypeId = '/' + event.newVal.get('identifier');
+
+            Y.io(this.get('host').get('app').get('apiRoot') + 'api/ezp/v2/contentonthefly/locations' + contentTypeId, {
```

Comment:
Nitpick: It is acceptable to make a variable called `requestUrl` and set `this.get('host').get('app').get('apiRoot') + 'api/ezp/v2/contentonthefly/locations' + contentTypeId` as its value.